src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategyGuarantee.kt

```diff
@@ -75,5 +75,10 @@ class ManagedStrategyGuarantee private constructor(
          * specified method calls.
          */
         fun treatAsAtomic() = ManagedStrategyGuarantee(classPredicate, methodPredicate, AnalysisSectionType.ATOMIC)
+
+        /**
+         * The methods will be treated as SILENT, so that no switch points will be placed unless forced.
+         */
+        internal fun mute() = ManagedStrategyGuarantee(classPredicate, methodPredicate, AnalysisSectionType.SILENT)
```

Comment:
I actually like the `mute` name!

We might consider renaming `SILENT` to `MUTED` (just an idea for the future, not for this PR).